Frontend/ui-customer/src/Screens/RentalBookingFlow/RentalScreen/View.purs

```diff
@@ -0,0 +1,308 @@
+          [ height WRAP_CONTENT
+          , width MATCH_PARENT
+          , margin $ Margin 16 0 16 32
+          , orientation VERTICAL
+          ]
+          ( map
+            ( \item ->
+                ChooseVehicle.view (push <<< ChooseVehicleAC) item
+            )
+            state.data.quoteList
+          )
+        ]
+    , linearLayout
+      [ height MATCH_PARENT
+      , width MATCH_PARENT
+      , gravity BOTTOM
+      ]
+      [ linearLayout
+        [ height WRAP_CONTENT
+        , stroke $ "1," <> Color.grey900
+        , padding $ Padding 16 16 16 16
+        ] [PrimaryButton.view (push <<< PrimaryButtonActionController) (primaryButtonConfig state)]
+      ]
+    ]
+
+fareBreakupView :: forall w. (Action -> Effect Unit) -> RentalScreenState -> PrestoDOM (Effect Unit) w
+fareBreakupView push state =
+  linearLayout
+    [ height MATCH_PARENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    , background Color.white900
+    ]
+      [ GenericHeader.view (push <<< GenericHeaderAC) (genericHeaderConfig state)
+      , separatorView push state
+      , linearLayout
+        [ height WRAP_CONTENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        , margin $ Margin 16 16 16 32
+        ] $ map ( 
+            \item -> if (item.index == item.activeIndex) then ChooseVehicle.view (\action -> pure unit) (item{showStroke = false, showInfo = false}) else dummyView 
+            ) state.data.quoteList
+      , linearLayout 
+        [ height WRAP_CONTENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        , margin $ MarginHorizontal 16 16
+        ]
+        (map (\item -> 
+          descriptionView push state (item)
+        ) [BookingTime, BookingDistance, BaseFare, TollFee])
+      , noteAndPrimaryButtonView push state
+      ]
+
+descriptionView :: forall w. (Action -> Effect Unit) -> RentalScreenState -> FareBreakupRowType -> PrestoDOM (Effect Unit) w
+descriptionView push state description =
+  linearLayout
+    [ height WRAP_CONTENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    ]
+    [ textView $
+      [ width MATCH_PARENT
+      , height WRAP_CONTENT
+      , text $ getTitleFromDescription description true
+      , color Color.black800
+      ] <> FontStyle.body1 TypoGraphy
+    , textView $
+      [ width MATCH_PARENT
+      , height WRAP_CONTENT
+      , text $ getTitleFromDescription description false
+      , color Color.black700
+      , margin $ MarginVertical 4 24
+      ] <> FontStyle.paragraphText TypoGraphy
+    ]
+  where 
+    getTitleFromDescription :: FareBreakupRowType -> Boolean -> String
+    getTitleFromDescription description toShowTitle = 
+      case description of
+        BookingTime -> if toShowTitle then getString BOOKING_ON <> state.data.rentalBookingData.selectedDate <> ", " <> state.data.rentalBookingData.selectedTime <> " (" <> (show state.data.rentalBookingData.baseDuration) <> "hrs)" else getString FINAL_FARE_DESCRIPTION
```

Comment:
(show state.data.rentalBookingData.baseDuration) put this in a variable and use it as it is being called more than once